src/main/java/org/zalando/nakadi/domain/EventTypeOptions.java

```diff
@@ -1,10 +1,7 @@
 package org.zalando.nakadi.domain;
 
-import javax.annotation.Nullable;
-
 public class EventTypeOptions {
 
-    @Nullable
     private Long retentionTime;
```

Comment:
Why not just add a `@NotNull` annotation and assign a default value in the constructor?

It would simplify the `getTopicRetentionMs` method. Since we would be sure that there is always a value there, we could remove all logic and just return the value. It also removes the need of a `settings` to hold default values.